packages/grpc-web/docker/jsunit-test/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM selenium/standalone-chrome:93.0
+
+# Node version in node:12-stretch image
+ARG NODE_VERSION=12.22.6
+
+USER root
+
+RUN apt-get update && \
+  apt-get install -y nodejs npm python
```

Comment:
Please make sure this installs `python3`.

Reply:
Good catch! Removed python here since the selenium image already has Python 3.8 installed :)